### PR TITLE
MF-196 create levy reservation

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: src/SFA.DAS.Reservations.Application.UnitTests/AccountReservation/Commands/CreateReservation/WhenValidatingTheCreateAccountReservationCommand.cs [hunk 1]
```diff
@@ -101,23 +105,61 @@ public async Task Then_The_Command_Is_Valid_If_Course_Exists()
...
+        [Test]
+        public async Task Then_If_The_Command_Is_For_A_Non_Levy_Reservation_The_Course_And_Start_Date_are_Not_Required()
+        {
+            //Act
+            var actual = await _validator.ValidateAsync(new CreateAccountReservationCommand
+            {
+                Id = Guid.NewGuid(),
+                AccountId = 5432,
+                AccountLegalEntityId = 1,
+                IsLevyAccount = true
```
Comment (scottywakefield) [inline src/SFA.DAS.Reservations.Application.UnitTests/AccountReservation/Commands/CreateReservation/WhenValidatingTheCreateAccountReservationCommand.cs]: title says non-levy

File: src/SFA.DAS.Reservations.Application.UnitTests/AccountReservation/Commands/CreateReservation/WhenValidatingTheCreateAccountReservationCommand.cs [hunk 2]
```diff
@@ -20,18 +20,19 @@ public void Arrange()
 
             _validator = new CreateAccountReservationValidator(_courseService.Object);
 
-            _courseService.Setup(s => s.GetCourseById(It.IsAny<string>()))
+            _courseService.Setup(s => s.GetCourseById(It.Is<string>(c=>c.Equals("1"))))
```
[the remark this answers is not in the document]
Comment (dashton82) [reply src/SFA.DAS.Reservations.Application.UnitTests/AccountReservation/Commands/CreateReservation/WhenValidatingTheCreateAccountReservationCommand.cs]: done - not sure why the above was there!

File: src/SFA.DAS.Reservations.Application.UnitTests/AccountReservation/Commands/CreateReservation/WhenValidatingTheCreateAccountReservationCommand.cs (hunk 1 above)
Comment (dashton82) [reply src/SFA.DAS.Reservations.Application.UnitTests/AccountReservation/Commands/CreateReservation/WhenValidatingTheCreateAccountReservationCommand.cs]: Updated test title

**Comment by robinoconnor** [comment]
Checked the latest update and all seems fine so as marking as resolved

**Pull request merged**